n_wrench_system_base:
  ros__parameters:
    controller: "ds4"

    wrench_topic_name: "joy_wrench"
    wrench_stamped_topic_name: "joy_wrench_stmp"

    send_stamped: True
    frame_id: base_link

    max_norm: 1.0

    equalization_type: "inv_cube"

    ds4_force_x: axis_left_y
    ds4_force_y: axis_left_x
    ds4_force_z: axis_right_y

    ds4_torque_x: axis_right_x
    ds4_torque_y: None
    ds4_torque_z: None